src/components/org/apache/jmeter/visualizers/backend/BackendListenerGui.java

```diff
@@ -71,6 +72,10 @@
 
     /** A panel allowing the user to set arguments for this test. */
     private ArgumentsPanel argsPanel;
+    
+    /** The current className of the Backend listenenr **/
+    private String className;
```

Comment:
This part does not seem to be related to this PR , it's probably here to avoid losing configuration when changing class. Would it be possible to make another PR ?

Reply:
Not only, for me these changes are an integral part of the PR. Without it, you can swap class without lose or merge configurations.

Reply:
ok